xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-plugins/src/main/webjar/xwiki-resource/resourcePicker.js

```diff
@@ -48,6 +50,8 @@ define('resourcePicker', [
     options = options || {};
 
     var resourcePicker = $(resourcePickerTemplate);
+    resourcePicker.find('button.dropdown-toggle').first().attr('title', translations.get('dropdown.toggle.title'));
+    resourcePicker.find('button.dropdown-toggle .sr-only').first().text(translations.get('dropdown.toggle.title'));
```

Comment:
```suggestion
    resourcePicker.find('button.dropdown-toggle').attr('title', translations.get('dropdown.toggle.title'));
    resourcePicker.find('button.dropdown-toggle .sr-only').text(translations.get('dropdown.toggle.title'));
```

No need to limit the collection to the first item.